pkg/minikube/driver/driver.go

```diff
@@ -141,17 +145,69 @@ func IsDocker(name string) bool {
 	return name == Docker
 }
 
-// IsDockerDesktop checks if the driver is a Docker for Desktop (Docker on windows or MacOs)
+func IsLinuxDockerDesktop(name string) bool {
+	if IsDocker(name) {
+		cmd := exec.Command("docker", "version")
+		stdout, err := cmd.StdoutPipe()
+		if err != nil {
+			return false
```

Comment:
We generally want to log errors as this can help with debugging when users report issues.

For example:
```
if err != nil {
    klog.Warningf("failed to get docker version: %v", err)
    return false
}
```
The error isn't surfaced to the user, but it's saved to the log file.